src/TxHistory/TxHistory.stories.tsx

```diff
@@ -13,6 +14,21 @@ storiesOf('V2/TxHistory', module)
       </SelectedWalletProvider>
     )
   })
+  .add('default with API errors', () => {
+    const wallet = {
+      ...mockWallet,
+      fetchCurrentPrice: () => Promise.reject(new Error('fetchCurrentPrice failed')),
+      fetchUTXOs: () => Promise.reject(new Error('fetchUTXOs failed')),
```

Comment:
Since the Boundary is using `intl` you will need to add the i18n decorator as well.

Reply:
LanguageProvider wraps the entire app

Reply:
Is the question related to consider the cases where the errors messages are generated with `react-intl` @stackchain ?